test/integration/targets/inventory_docker_machine/runme.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# set DIGITALOCEAN_TOKEN to your Digital Ocean API token before running this test
+
+export DM_MACHINE_NAME=dm-test-machine
+
+# restrict Ansible just to our inventory plugin, to prevent inventory data being matched by the test but being provided
+# by some other dynamic inventory provider
+export ANSIBLE_INVENTORY_ENABLED=docker_machine
+
+[[ -n "$DEBUG" || -n "$ANSIBLE_DEBUG" ]] && set -x
+
+set -euo pipefail
+
+cleanup() {
+    echo "Cleanup"
+    ansible-playbook -i inventory_1.docker_machine.yml playbooks/teardown.yml
+    echo "Done"
+    exit 0
+}
+
+trap cleanup INT TERM EXIT
+
+echo "Check preconditions"
+# Host should NOT be known to Ansible before the test starts
+! ansible-inventory -i inventory_1.docker_machine.yml --list --host ${DM_MACHINE_NAME}
```

Comment:
This doesn't seem to work: `ERROR! Conflicting options used, only one of --host, --graph or --list can be used at the same time.`

Reply:
Fixed.